.github/workflows/testing.yml

```diff
@@ -0,0 +1,134 @@
+name: Pre testing
```

Comment:
Are you intending to merge this workflow as well or is this a leftover from testing?

Reply:
sorry it's a remnant of the tests, I'll delete it